web/styles/zulip.css

```diff
@@ -3606,3 +3606,49 @@ select.invite-as {
         border: 1px solid var(--color-border-personal-menu-avatar);
     }
 }
+
+.typing-animation {
+    display: inline-flex;
+    align-items: baseline;
+    margin-left: 3px;
+    width: 40px;
+    height: 5px;
+}
+
+.dot {
+    width: 4px;
+    height: 4px;
+    background-color: hsl(0deg 0% 100%);
+    border-radius: 50%;
+    margin-left: 3px;
+    opacity: 0;
+}
+
+.dot:nth-child(1) {
+    animation: typing 1s infinite 0.2s;
+}
+
+.dot:nth-child(2) {
+    animation: typing 1s infinite 0.4s;
+}
+
+.dot:nth-child(3) {
+    animation: typing 1s infinite 0.6s;
+}
```

Comment:
These CSS rules belong in `message_row.css`, since that's where we define CSS for the internals of messages. 

You should also give a less generic name than `dot`, and use SCSS scoping to nest the `nth-child` rules inside the base rule, etc. Look at how other message row CSS rules are defined.

Reply:
updated!